### ISSUE TITLE
Bug - v3.0.3 fails to run

### ISSUE THREAD

**Issue by abarbalat** [Guimove/terraform-aws-bastion#158]
It seems that the new output variable 'bastion_elb_id' is not compatible with optional load balancer

```
 Error: Missing resource instance key

value       = aws_lb.bastion_lb.id
 Because aws_lb.bastion_lb has "count" set, its attributes must be accessed on specific instances.

For example, to correlate with indices of a referring resource, use:
│     aws_lb.bastion_lb[count.index]
```

**Comment by mpoitevineau-millin** [comment]
Same on our side. Impossible to use the module since the 3.0.3

**Comment by Guimove** [comment]
Should be fixed now in v3.0.5
